lib/rules/web/admin_console/4.14/monitoring_alerts.xyaml

```diff
@@ -88,7 +88,7 @@ perform_silence:
 check_silence_detail:
   element:
     selector:
-      xpath: //span[text()='Silence details']
+      xpath: //h2[@class='co-section-heading' and text()='Silence details']
```

Comment:
If use `//h2[text()='Silence details']` can identify the element, better use this one. class attribute easy to change.